plugins/log/src/lib.rs

```diff
@@ -83,6 +85,24 @@ pub enum RotationStrategy {
     KeepOne,
 }
 
+#[derive(Debug, Clone)]
+pub enum TimezoneStrategy {
+    UseUtc,
+    UseLocal,
+}
+
+impl TimezoneStrategy {
+    pub fn get_now(&self) -> OffsetDateTime {
+        match self {
+            TimezoneStrategy::UseUtc => OffsetDateTime::now_utc(),
+            TimezoneStrategy::UseLocal => match OffsetDateTime::now_local() {
+                Ok(v) => v,
+                Err(_) => OffsetDateTime::now_utc() // Fallback to UTC since Rust cannot determine local timezone
+            }
```

Comment:
I think this could be simplified to something like this
```suggestion
            TimezoneStrategy::UseLocal => OffsetDateTime::now_local().unwrap_or_else(|_| OffsetDateTime::now_utc()) // Fallback to UTC since Rust cannot determine local timezone
```

Reply:
Aaand please run `cargo fmt` so that CI is happy again